client/src/index.css

```diff
@@ -5,9 +5,91 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
```

Comment:
Try to use material ui styling (we have some examples in the starter files) instead of plain css, it will make things more manageable and reusable later on